src/Compilers/CSharp/Portable/Lowering/ClosureConversion/SynthesizedClosureMethod.cs

```diff
@@ -168,7 +168,7 @@ protected override ImmutableArray<TypeSymbol> ExtraSynthesizedRefParameters
             => ImmutableArray<TypeSymbol>.CastUp(_structEnvironments);
         internal int ExtraSynthesizedParameterCount => this._structEnvironments.IsDefault ? 0 : this._structEnvironments.Length;
 
-        internal override bool InheritsBaseMethodAttributes => BaseMethod is LocalFunctionSymbol;
+        internal override bool InheritsBaseMethodAttributes => true;
```

Comment:
I'm not very familiar. What's a scenario affected by this change?

Reply:
> What's a scenario affected by this change?

For rewritten lambdas or local functions (rewritten during lowering by `ClosureConversion`), this property ensures any parameters in the rewritten method refer back to the original lambda or local function property for attributes. That is required to emit the attributes.

---
In reply to: [607479101](https://github.com/dotnet/roslyn/pull/52178#discussion_r607479101) [](ancestors = 607479101)